dog.py

```diff
@@ -437,6 +473,40 @@ def generate_env_arg_list(config: DogConfig) -> List[str]:
     return args
 
 
+def docker_run_volumes_from(config: DogConfig):
+    cmd = docker_cmd(config)
+
+    import asyncio
+
+    async def run_volume_installer(name, image):
+
+        proc = await asyncio.create_subprocess_exec(
+            cmd,
+            'run',
+            '--network',
+            'none',
+            '--name',
+            name,
+            image,
+            stderr=asyncio.subprocess.DEVNULL,
+        )
+        return await proc.wait()
+
+    loop = asyncio.get_event_loop()
+    tasks = [
+        run_volume_installer(name, image)
+        for name, image in config[VOLUMES_FROM].items()
+    ]
+    exit_codes = loop.run_until_complete(asyncio.gather(*tasks))
+    for name, ec in zip(config[VOLUMES_FROM].keys(), exit_codes):
+        if not (ec == 0 or ec == 125):
```

Comment:
Can i get a comment about what exit code 125 means?

Reply:
Yes (forgot that) - 125 is when the container (with this name) already exists. I'm actually leaning towards not checking the error code as 125 is a little too implementation specific for me - and I have not tested that it's the same using podman.